packetbeat/protos/tls/_meta/fields.yml

```diff
@@ -352,3 +352,351 @@
             type: keyword
             description: >
               An array containing the TLS alert type for every alert received.
+
+    #
+    # Aliases from pre-ECS fields to tls.detailed
+    #
+
+    # Replaced fields
+
+    - name: tls.handshake_completed
+      type: alias
+      path: tls.established
+      migration: true
```

Comment:
Do these fields show up in `filebeat export template` by default? I'm just wondering what effect `migration: true` has? Or does `migration.6_to_7.enabled: true` need to be set for them to show up (I don't think we want that to be needed).

Reply:
You're right. I removed it and now they show up without the 6 to 7 flag.